R/Gargle-class.R

```diff
@@ -141,15 +141,21 @@ Gargle2.0 <- R6::R6Class("Gargle2.0", inherit = httr::Token2.0, list(
     # append the email
     paste(msg, self$email, sep = "-")
   },
+  cache = function(path = self$cache_path) {
+    "!DEBUG cache a token"
+    cache_token(self, path)
+    self
+  },
```

Comment:
This is same as that in `Token` class in httr, but seems needed in here so that `gargle::cache_token()` is used instead of `httr::cache_token()`.

Reply:
Maybe call it `add_token_to_cache()`?

Reply:
Or maybe `token_cache()`

Reply:
renamed to `token_cache()`